mortar/src/main/java/mortar/RealScope.java

```diff
@@ -33,52 +33,41 @@
   protected boolean dead;
 
   private final Set<Scoped> tearDowns = new HashSet<>();
-  private final Object graph;
-  private final RealScope parent;
+  final RealScope parent;
   private final String name;
+  private final Map<String, Object> services;
 
-  RealScope(Object objectGraph) {
-    this(MortarScope.ROOT_NAME, null, objectGraph);
-  }
-
-  RealScope(String name, RealScope parent, Object graph) {
-    this.graph = graph;
+  RealScope(String name, RealScope parent, Map<String, Object> services) {
```

Comment:
Do we still need RealScope vs Scope? There's no subclass any more, so you can make everything private or package protected in here and get rid of the interface.